iothub/device/src/Exceptions/IotHubStatusCode.cs

```diff
@@ -62,6 +62,12 @@ public enum IotHubStatusCode
         /// </remark>
         Suspended,
 
+        /// <summary>
+        /// The request failed because the operation times out which can be caused by underlying network issues 
```

Comment:
```csharp
        /// The request failed because the operation timed. This can be caused by underlying network issues 
        /// or by the server being too busy to handle the request.
```